Source/WebKit/UIProcess/ios/DragDropInteractionState.mm

```diff
@@ -314,9 +314,9 @@ static bool canUpdatePreviewForActiveDragSource(const DragSourceState& source)
     }
 
     if (shouldUseTextIndicatorToCreatePreviewForDragSource(source)) {
-        auto indicator = source.indicatorData.value();
-        auto textIndicatorImage = uiImageForImage(indicator.contentImage.get());
-        return createTargetedDragPreview(textIndicatorImage.get(), contentView, previewContainer, indicator.textBoundingRectInRootViewCoordinates, indicator.textRectsInBoundingRectCoordinates, cocoaColor(indicator.estimatedBackgroundColor).get(), nil, addPreviewViewToContainer).autorelease();
+        auto indicatorData = source.indicatorData.value();
+        auto textIndicatorImage = uiImageForImage(indicatorData.contentImage.get());
```

Comment:
```suggestion
        RetainPtr textIndicatorImage = uiImageForImage(indicatorData.contentImage.get());
```

Reply:
Can't drop indicatorData, it's used extensively below ;P